README.md

```diff
@@ -23,6 +23,9 @@ $ sudo apt-get install ros-indigo-actionlib ros-indigo-geometry-msgs ros-indigo-
 
 Once ROS is installed, you can install AIKIDO from the [Personal Robotics Lab PPA](https://launchpad.net/~personalrobotics/+archive/ubuntu/ppa):
 ```shell
+$ sudo apt-add-repository ppa:libccd-debs/ppa
+$ sudo apt-add-repository ppa:fcl-debs/ppa
+$ sudo apt-add-repository ppa:dartsim/ppa
```

Comment:
Wait, these lines use `*apt*-add-repository`,
but the next line uses `add-*apt*-repository`?

Which one is it?

Reply:
Technically, [they are interchangeable](https://askubuntu.com/a/83490). I think we should be consistent at least in the same text. Let me change these to `add-apt-repository`.